src/main/scala/lms/thirdparty/cuda/cuda.scala

```diff
@@ -1065,9 +1068,13 @@ trait CudaLibs extends CudaOps {
     cudaSyncThreads
   }
 
-  def softmax[N:Numeric:Manifest](log: Boolean)(implicit  __pos: SourceContext) = cudaGlobalDynamicFun {
+  def softmax[N:Numeric:Manifest](logSoftmax: Boolean)(implicit __pos: SourceContext) = cudaGlobalDynamicFun {
```

Comment:
maybe use `cudaSoftmax`, `cudaSoftmaxGrad` to keep naming consistency with other kernels?

Reply:
yep, makes sense! will update. Thanks!